lib/features/push_notification/presentation/action/fcm_action.dart

```diff
@@ -0,0 +1,13 @@
+
+import 'package:jmap_dart_client/jmap/account_id.dart';
+import 'package:tmail_ui_user/features/base/action/ui_action.dart';
+
+class SynchronizeEmailAction extends FcmAction {
```

Comment:
we will not synchronize email in background. When user start the app again, we will
- open from notification
- open from apps
Of course, it is not in the scope of this ticket.

Reply:
> we will not synchronize email in background. When user start the app again, we will
> 
> * open from notification
> * open from apps
>   Of course, it is not in the scope of this ticket.

So you have to save this received message. If the case it doesn't have `type=EmailDelivery` it won't have notification. Do you have any solution yet?